rmgpy/chemkin.pyx

```diff
@@ -1227,7 +1237,7 @@ def read_thermo_block(f, species_dict):
                     logging.warning('Skipping unexpected species "{0}" while reading '
                                     'thermodynamics entry.'.format(label))
             thermo_block = ''
-        if len(thermo_block.split('/n')) > 4:
+        if len(thermo_block.split('/n')) > 5:
             raise ChemkinError('Should only have 4 lines in a thermo block:\n{0}'.format(thermo_block))
```

Comment:
nitpicky, but maybe change error message text: 
```suggestion
            raise ChemkinError('Should only have a maximum of 5  lines in a thermo block:\n{0}'.format(thermo_block))
```

Reply:
Thanks Chris! Just fixed this